lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -77,25 +78,41 @@ $description = Get-AnsibleParam -obj $params -name "description" -type "str" -de
 $enabled = Get-AnsibleParam -obj $params -name "enabled" -type "bool" -default $true
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -default '\'
 
+# Ensure the TaskPath starts with and ends with slashes as this causes issues with some
+# versions of Windows
+if ($path[0] -ne '\') {
```

Comment:
I would rather fail and return a message back to the user. Because we cannot assume what the user wants. Adding a trailing slash is no problem, but prepending is not a good idea IMO